lib/batchVersions.js

```diff
@@ -5,6 +5,11 @@ import * as bedrock from 'bedrock';
 import * as database from 'bedrock-mongodb';
 import {promisify} from 'util';
 const {util: {BedrockError}} = bedrock;
+import LRU from 'lru-cache';
+const CACHE = new LRU({
+  // FIXME: what is appropriate size and maxAge?
+  max: 100,
```

Comment:
We could make max age 24 hours just to be safe for some carefully crafted hack we might have to do in the future, but really, versions should never change or the system is going to break.

```suggestion
  max: 10,
```

I don't think we should have more than 10 versions in hot paths at any time -- and 10 already seems excessive.